File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/ThanhVienService.cs
namespace Service.Implement
{
    public class ThanhVienService : BaseService<ThanhVien, IThanhVienRepository>
    , IThanhVienService
    {
    private readonly IThanhVienRepository _ThanhVienRepository;
    public ThanhVienService(IThanhVienRepository ThanhVienRepository) : base(ThanhVienRepository)
    {
    _ThanhVienRepository = ThanhVienRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/ThanhVien.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ThanhVien } from 'src/app/shared/ThanhVien.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class ThanhVienService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "ThanhVien";
    }
}

